README.md

```diff
@@ -46,6 +46,33 @@ sudo podman run \
     quay.io/centos-bootc/fedora-bootc:eln
 ```
 
+### Using local containers
+
+To use containers from local container's storage rather than a registry, we need to ensure two things:
+- the container exists in local storage
```

Comment:
I guess I'm ok with this constraint, but in the podman-bootc CLI we intentionally matched `podman run` here and do "pull if not present" (which matches `docker run` of course).